pkg/playground/plot-pf4.css

```diff
@@ -0,0 +1,35 @@
+.sparky {
```

Comment:
Could you please add a comment why we have to do that and where you got the CSS from?

We might eventually move this to pkg/lib/, but let's do that when we rewrite the graphs page.